test/parallel/test-process-cpuUsage.js

```diff
@@ -33,27 +33,57 @@ for (let i = 0; i < 10; i++) {
   assert(diffUsage.system >= 0);
 }
 
+const invalidUserArgument = new RegExp('^TypeError: value of user ' +
```

Comment:
I think 4 (i.e. an extra indent to separate it from any indented code on the next line)